zproject/sentry.py

```diff
@@ -50,6 +58,7 @@ def setup_sentry(dsn: Optional[str], *integrations: Integration) -> None:
             *integrations,
         ],
         before_send=add_context,
+        send_default_pii=True,
```

Comment:
I added a block comment above this line containing some of the information from the commit message, since it seems like a useful reference for posterity.